src/components/POLine/const.js

```diff
@@ -13,6 +13,7 @@ export const ACCORDION_ID = {
   itemDetails: 'itemDetails',
   lineDetails: 'lineDetails',
   location: 'location',
+  other: 'other',
```

Comment:
I think it make sense to add more mappings due to required fields below, in `export const MAP_FIELD_ACCORDION = {`

Reply:
something like `physical: ACCORDION_ID.physical,` 
but even after that there might be a problem with other accordion expanding on validation error when material type is required